src/formatDate.js

```diff
@@ -8,7 +8,35 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  // eslint-disable-next-line max-len
+  const dateParts = date.split(fromFormat[3]); // Використовуємо роздільник з fromFormat
```

Comment:
Consider using destructuring to assign values to variables representing the date parts for better readability. For example, `[year, month, day] = date.split(fromFormat[3]);`